### ISSUE TITLE
obackup does not create a backup for me

### ISSUE THREAD

**Issue by wenlzhang** [chrisgrieser/shimmering-obsidian#12]
I tried to use `obackup` to create a backup of my vault. However, no backup was created in the specified folder, although I got the notification saying that the backup was successfully created.

**Comment by chrisgrieser** [comment]
Hi! please follow the Instructions in the [Trouble Shooting Section](https://github.com/chrisgrieser/shimmering-obsidian#troubleshooting). If the issue still persists, I will need a debugging log and information on your Mac OS Version, and whether you are on an M1.

Please also note that version 1.6 has fixed a bug with `obackup`, so if by any chance you are on an older version of the workflow,[ you can simply update](https://github.com/chrisgrieser/shimmering-obsidian/releases).

**Comment by wenlzhang** [comment]
Hi,

Thanks for the prompt response!

I have gone through the Trouble Shooting section but did not resolve the issue.

My macOS version is Big Sur Version 11.5.2, and it is an Intel-chip Mac. The debugging log is as follows:
```
[22:35:57.076] Logging Started...
[22:35:59.967] Shimmering Obsidian[Script Filter] Queuing argument '(null)'
[22:35:59.972] Shimmering Obsidian[Script Filter] Queuing argument '(null)'
[22:36:01.439] Shimmering Obsidian[Keyword] Processing complete
[22:36:01.440] Shimmering Obsidian[Keyword] Passing output '' to Post Notification
[22:36:01.440] Shimmering Obsidian[Post Notification] Processing complete
[22:36:01.441] Shimmering Obsidian[Post Notification] Passing output '' to Run Script
[22:36:01.441] STDERR: Shimmering Obsidian[Run Script] /.../Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.D29ABBD5-F375-4C89-B556-DF4C2B2E5309/obsidian-backup.sh:13: no matches found: ./.trash/*
[22:36:01.442] Shimmering Obsidian[Run Script] Processing complete
[22:36:01.443] Shimmering Obsidian[Run Script] Passing output '/.../ObsidianVaultBackup/Obsidian-Backup_2021-09-04_22-36.zip
' to Post Notification
[22:36:01.443] Shimmering Obsidian[Post Notification] Processing complete
[22:36:01.444] Shimmering Obsidian[Post Notification] Passing output '/.../ObsidianVaultBackup/Obsidian-Backup_2021-09-04_22-36.zip
' to Play Sound
[22:36:01.513] Shimmering Obsidian[Play Sound] Processing complete
[22:36:01.514] Shimmering Obsidian[Play Sound] Passing output '/.../ObsidianVaultBackup/Obsidian-Backup_2021-09-04_22-36.zip
' to Call External Trigger
[22:36:01.515] Shimmering Obsidian[External] Processing complete
[22:36:01.516] Shimmering Obsidian[External] Passing output '/.../ObsidianVaultBackup/Obsidian-Backup_2021-09-04_22-36.zip
' to Run Script
```

**Comment by chrisgrieser** [comment]
Got it. Was only a minor issue, has been fixed in [release 1.7.3](https://github.com/chrisgrieser/shimmering-obsidian/releases/tag/1.7.3). :)
